st3/sublime_lib/resource_path.py

```diff
@@ -11,6 +11,12 @@
 __all__ = ['ResourcePath']
 
 
+RESOURCE_ROOTS = {
+    'Packages': sublime.packages_path(),
```

Comment:
This is not good. Users will usually import `sublime_lib` when their module is loaded, meaning this is potentially (or likely, even) evaluated before the API has loaded and yields `None`.

Reply:
Fixed.